src/lib/style-properties.html

```diff
@@ -285,6 +285,13 @@
         return {properties: props, key: o};
       },
 
+      _rootSelector: /(?:^:root)|(?::host\s*>\s*\*)/,
```

Comment:
Yeah, there's no need for `:root` to be first in the selector, removed with 63433c8